data_relay.py

```diff
@@ -8,9 +8,10 @@
 from comm_server import TelemetryFactory, ProducerToManyClient
 from telem_producer import TelemetryProducer
 
-db_type = (
+db_type = [(
```

Comment:
I would do a map and keeping the bi-map of struct type name -> struct type encoding. That will keep you flexible long run.